Material.Ripple/RippleEffect.cs

```diff
@@ -57,30 +56,41 @@ protected override void OnSizeChanged(SizeChangedEventArgs e) {
         }
 
         private void PointerPressedHandler(object? sender, PointerPressedEventArgs e) {
+            var c = _container;
+
+            if (c is null)
+                return;
+
             var (x, y) = e.GetPosition(this);
-            if (_container is null || x < 0 || x > Bounds.Width || y < 0 || y > Bounds.Height) {
+
+            if (x < 0 || x > Bounds.Width || y < 0 || y > Bounds.Height)
                 return;
-            }
+
             _isCancelled = false;
 
             if (!IsAllowedRaiseRipple)
                 return;
 
+            CreateRippleInstancePrivate(c, x, y);
```

Comment:
Probably we should remove Private from method name?

Reply:
this is a private function since i have such naming preferences recently

Reply:
but i can modify it if you cant accept it

Reply:
I don't mean any name, so you can pick any variant